src/commands/extractText.ts

```diff
@@ -1,20 +1,27 @@
 import { commands, window, workspace } from 'vscode'
-import { trim } from 'lodash'
 // @ts-ignore
-import * as limax from 'limax'
 import { ExtensionModule } from '../modules'
 import { Commands, Config, CurrentFile } from '../core'
 import i18n from '../i18n'
 import { ExtractTextOptions } from '../editor/extract'
 import { promptTemplates } from '../utils'
 import { overrideConfirm } from './overrideConfirm'
 import { keypathValidate } from './keypathValidate'
+import * as limax from 'limax'
+import { trim } from 'lodash'
 
 const m: ExtensionModule = () => {
   return commands.registerCommand(Commands.extract_text,
     async(options: ExtractTextOptions) => {
       const { filepath, text, range, languageId } = options
-      const default_keypath = limax(text, { separator: Config.preferredDelimiter, tone: false }) as string
+      let default_keypath: string
+      const keygenStrategy = Config.keygenStrategy
+      const keyPrefix = Config.keyPrefix
+      if (keygenStrategy === 'random')
+        default_keypath = `${keyPrefix}t${Date.now()}`
```

Comment:
`Date.now()` is not that random I think. You can use `nanoid()` here since it's already a dependency.